### DISCUSSION TITLE
class 'MongoDB\Database' does not have a method 'getDoctrineDriver'

### DISCUSSION THREAD

**Discussion by MeerKatDev** [jenssegers/laravel-mongodb#2221] [Q&A] [answered]
- Laravel-mongodb Version: 3.8
- PHP Version: 7.4
- Database Driver & Version: 
```
pro@MBP-Macbook lookinglass-backend % mongo --version
MongoDB shell version v4.4.3
Build Info: {
    "version": "4.4.3",
    "gitVersion": "913d6b62acfbb344dde1b116f4161360acd8fd13",
    "modules": [],
    "allocator": "system",
    "environment": {
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}
```

### Description:

I'm trying to use this package with Backpack. Backpack is requesting that method:
```
 public function getDoctrineConnection()

    {

        if (is_null($this->doctrineConnection)) {

            $driver = $this->getDoctrineDriver();
 ...

```

As far as I understood, this is an issue with the fact that now Backpack uses the doctrine abstraction to access the db. I'm happy to work on it, I'm just unsure on where I should start, because there seems to be no way to get hold of the driver object in mongodb.

**Comment by divine** [comment, marked as answer]
Hello,

This isn't an issue with this library.

Thanks!

**Reply by MeerKatDev** [reply]
According to https://github.com/Laravel-Backpack/CRUD/issues/2883 , it's not an issue with the other library either, so it's difficult to understand where to open the issue.

**Reply by divine** [reply]
Hello,

It's definitely an issue with that library and looks like they're not willing to fix it.

See https://github.com/Laravel-Backpack/CRUD/pull/1891 and https://github.com/Laravel-Backpack/CRUD/issues/802

Thanks!
